Specs/NumberWithUnit/French/CurrencyModel.json

```diff
@@ -29,11 +29,11 @@
 }
 ,
 {
-  "Input": "Siegel payé national et 94.000 $ shuster pour laisser tomber toutes les réclamations.",
+  "Input": "Siegel payé national et 94000 $ shuster pour laisser tomber toutes les réclamations.",
   "NotSupportedByDesign": "python",
   "Results": [
     {
```

Comment:
If 94.000 is a valid number format in French, we should also support it. 
If it is, let's also open another issue to implement support for this and create a new unit test case for this example, ok?

Reply:
Most common use is 'spaces' to separate thousands, although period is formally accepted - [see source](https://french.kwiziq.com/revision/grammar/how-to-write-decimal-numbers-in-french). It should basically be the same as the spanish one. When I added a 'space' case to the spanish recognizer it did not pass. The most important is that commas denote decimal separator